governance/polls/ MIP9 Community Greenlight Poll - [OUSD] - January 10, 2022.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Community Greenlight Poll - [OUSD] - January 10, 2022
```

Comment:
Template is "$asset_code ($asset_name)". Remember, everything with a $ sign is something you need to change. So this should be "Community Greenlight Poll - OUSD (Origin Dollar) - January 10, 2022", same in the file name